aries_cloudagent/vc/vc_ld/store.py

```diff
@@ -0,0 +1,42 @@
+"""Verifiable Credential storage methods."""
+from ...admin.request_context import AdminRequestContext
+from ...storage.vc_holder.base import VCHolder
+from ...storage.vc_holder.vc_record import VCRecord
+from ...vc.vc_ld import VerifiableCredential
+from ...vc.ld_proofs import DocumentLoader
+from pyld import jsonld
+from pyld.jsonld import JsonLdProcessor
+
+
+async def store_credential(
```

Comment:
Should this belong as a method of the VcLdpManager?

Reply:
I think so, looking at the other actions such as verify/issue/etc.., they have their own files and a method calling it from the VcLdpManager, I would lean towards keeping this file and adding a method calling this function to keep the same model?

Reply:
Grabbed a few links for context/reference:
- Issue: https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/vc/vc_ld/issue.py
- Verify: https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/vc/vc_ld/verify.py
- Prove: https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/vc/vc_ld/prove.py

I would categorize these as primitives that are stateless and encapsulate the inner workings of the LD Proofs package.

Storage, on the other hand is stateful, as it records the credential in the wallet. The manager already operates in this sphere (e.g. https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/vc/vc_ld/manager.py#L92-L116) so I think I put my vote down for having storage of a credential be managed by the `VcLdpManager`.

Reply:
after looking at the code some more, this is also what I landed on